operators/grpc_operators/python/server/grpc_server_response.py

```diff
@@ -0,0 +1,42 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+from holoscan.core import IOSpec, Operator, OperatorSpec
+
+from holohub.grpc_operators import holoscan_pb2 as holoscan_proto
+from operators.grpc_operators.python.common.tensor_proto import TensorProto
+
+
+class GrpcServerResponseOp(Operator):
+    def __init__(self, fragment, response_queue, *args, **kwargs):
```

Comment:
please add type hint for `response_queue`

Reply:
Added type hints